packages/x-components/src/wiring/wiring.types.ts

```diff
@@ -30,7 +30,7 @@ export type Wire<PayloadType> = (
  */
 export interface WireMetadata {
   /** The {@link QueryFeature} that originated the event. */
-  feature?: QueryFeature;
+  feature?: QueryFeature | ResultFeature;
```

Comment:
Don't know of any, maybe in the future

Reply:
Maybe we should ask Gerardo about this, just in case.
If this can happen, then we can split this in two properties, something like `feature` and `resultFeature`.